### PR TITLE
#167709673: Updating html Home Page

### PR DESCRIPTION
**Pull request by Niyonsengaeric**
#### What does this PR do?
 - Editing Home Page
#### Description of Task to be completed?
*Home page*
- The files inside the folder UI folder should be in lower case
- The Menu bar should be arranged with a search on it
- The footer should be Fixed
- Na empty space or Unwanted comments
- The photo Should be arrange 2-2 on tablet then 4 on the desktop
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167709673

#### Screenshots (if appropriate)

home page change
home page change2
home page change3

### CODE REVIEW

**Comment by oesukam** [comment]
@Niyonsengaeric Cards under *Latest News* and *Popular Destinations* are not yet centred

**Comment by oesukam** [comment]
@Niyonsengaeric Why are tests failing?

**Review by oesukam — CHANGES_REQUESTED** [review]
File names should not have white space to handle cross platforms compatibility such as `bus 1.jpg` should `bus-1.jpg` or `bus_1.jpg`

File: UI/index.html [hunk 1]
```diff
@@ -1,179 +1,161 @@
...
+            <h2>Ticket of Your Desired Destinations</h2>
+            <h3>Cheap and Easy</h3>
+            <p>
+              We Offer You Tickets At Good Price, We also Provide a good
+              Services to our client any time and and any where in our
+              Destinations
+            </p>
+
+            <br />
+
```
Comment (oesukam) [inline UI/index.html:58]: Why this space, I think it is not necessary?

**Review by oesukam — CHANGES_REQUESTED** [review]

File: UI/index.html (hunk 1 above)
Comment (Niyonsengaeric) [reply UI/index.html:58]: on the issue of tests am not seeing why it fails but in my local machine it works if there is any help you may help me.

Comment (Niyonsengaeric) [reply UI/index.html:58]: on the problem of spacing  am going to remove it

Comment (oesukam) [reply UI/index.html:58]: You need to check your Travis settings for tests but you keep it for now and work on it when you start applying the backend's feedback

**Pull request closed without merging**